CIPs/0019.md

```diff
@@ -0,0 +1,72 @@
+---
+cip: 0019
+title: JWT signer for DID-JWT
+author: @daoauth
+discussions-to: https://github.com/celo-org/celo-proposals/issues/73
+status: Draft
+type: 
+category: 
+created: 2020-09-09
+license: Apache 2.0
+---
+
+## Simple Summary
+JWT encryption is required for web and did support.
+
+## Abstract
```

Comment:
I'd like a definition of did-jwt or a link to it for clarity

Reply:
did-jwt definitions have been updated.